### ISSUE TITLE
Allow drawing boardless panel? 

### ISSUE THREAD

**Issue by YongHee-Kim** [foldfelis/DisplayStructure.jl#29]
Hello 
Thanks for nice package. 

I've noticed that you already support borderless `Rectangle` but those option is not accessble from `Panel`

I've made this commit to pass `kwargs` to the `Rectangle` from the `Panel`
https://github.com/YongHee-Kim/DisplayStructure.jl/commit/87a56dd00968486f0814121aec52da95323aa3c6

here is MWE
```julia 
using DisplayStructure
DisplayStructure.Panel("TestPanel", [5,12],[1,1];background = '.', border = false) |> render
```

![image](https://user-images.githubusercontent.com/7087745/156346316-0637fc61-aee9-4898-8309-0937aa1d7dae.png)

**Comment by foldfelis** [comment]
Hi @YongHee-Kim , thanks for the MWE. I was planning to implement something like [rich](https://github.com/Textualize/rich) but with more dynamicly features. But currently I am too busy to make any progress on this project. Do you have any interest to collaberate to this project?

**Comment by YongHee-Kim** [comment]
Yes I'm interested in contributing to your project. While I'm not sure my knowledge on CS is good enough to be helpful (I'm struggling to understand the basic data types like `IOBuffer`), I will do what I can if you would guide me with the code 😃

**Comment by foldfelis** [comment]
Please feel free to open a PR and tag me to review
